app/assets/javascripts/getFamilies.js

```diff
@@ -0,0 +1,27 @@
+$(document).ready(function() {
+  getFamilymembers();
+});
+
+
+var getFamilymembers = function(){
+  $.ajax({
+    url: "http://localhost:3000/families.json",
+    type: 'get',
+    dataType: 'json'
+  }).done(function(response){
+    $('.familyname-container').html("");
+    var memberMedalTemplate = $('#memberMedal-template').html();
+    var dropdown = $('#child-points').html();
+    var familynameTemplate = $('#familyname-template').html();
+    var familynameRender = Mustache.render(familynameTemplate, response);
+    $('.familyname-container').append(familynameRender);
+    $('.members-medals-container').html("");
+    $('.child-points').html("");
+    response.members.forEach(function(member){
+      var memberMedalRender = Mustache.render(memberMedalTemplate, member);
```

Comment:
This feels like it should be in the view using a mustache "each" with your collection rather than having you loop through and repeatedly call render.

Also this whole section could do with refactoring. Even if you don't want to go full MVC, think about pulling out functions to do the repetitive work.